Support/Fuego/Mechanism/Models/dodecane_lu_qss/README.md

```diff
@@ -0,0 +1,34 @@
+# Analytically Reduced Mechanism
```

Comment:
Should the content of this file really belong in the actual documentation?

Reply:
In the documentation, we also explain what is mentioned here about the linearization methods. But I thought that it could be useful to reiterate here. Should I remove it from the Readme and redirect the user to the documentation?